docs/content/manual/release-specific/v1.1.0/prometheus_support.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Prometheus Support
+---
+Prometheus Support allows user to monitor the longhorn metrics. The details are available at https://longhorn.io/docs/1.1.0/monitoring/
+## Monitor longhorn
+1. Deploy the Prometheus-operator, ServiceMonitor pointing to longhorn-backend and Prometheus as mentioned in the doc.
+2. Create an ingress pointing to Prometheus service.
+3. Access the Prometheus web UI using the ingress created in the step 2.
+4. Select the metrics from below to monitor the longhorn resources.
+    1. longhorn_volume_capacity_bytes
+    2. longhorn_volume_actual_size_bytes
+    3. longhorn_volume_state
+    4. longhorn_volume_robustness
+    5. longhorn_instance_manager_cpu_requests_millicpu
+    6. longhorn_instance_manager_cpu_usage_millicpu
+    7. longhorn_instance_manager_memory_requests_bytes
+    8. longhorn_instance_manager_memory_usage_bytes
+    9. longhorn_manager_cpu_usage_millicpu
+    10. longhorn_manager_memory_usage_bytes
+    11. longhorn_node_count_total
+    12. longhorn_node_status
+    13. longhorn_node_cpu_capacity_millicpu
+    14. longhorn_node_cpu_usage_millicpu
+    15. longhorn_node_memory_capacity_bytes
+    16. longhorn_node_memory_usage_bytes
+5. Deploy workloads which use Longhorn volumes into the cluster. Verify that there is no abnormal data. e.g: volume capacity is 0, cpu usage is over 4000 milicpu etc.
+6. Attach a volume. Detach the volume. Verify that the volume's information is reported by at most 1 longhorn-manager at any time.
```

Comment:
Attach a volume. Detach the volume then attach it to a different node. Verify that the volume's information is reported by at most 1 longhorn-manager at any time.